src/LOAuth.ts

```diff
@@ -112,6 +112,42 @@ class LOAuth {
     return btoa(JSON.stringify(state));
   }
 
+  /**
+   * This method looks for a cookie set by PHC-Login that allows the
+   * applications to exchange auth tokens over different subdomains
+   * see phc-login/src/util/handleAuthResponse
+   */
+  private _getDomainCookieAuthState() {
+    try {
+      if (!document.cookie) {
+        return;
+      }
+      const cookie = document.cookie
+        .split('; ')
```

Comment:
Manual cookie parsing surprises me. Is there not a good cookie util in our apps?

Reply:
there's not that I could find, and this seemed simple enough, but maybe it's worth finding a lib?

Reply:
It's on the edge. I'm not sure if adding a new lib is worth it